### DISCUSSION TITLE
elastalert to TheHive alerter

### DISCUSSION THREAD

**Discussion by wdpless** [Security-Onion-Solutions/securityonion#1852] [Unsupported Versions] [answered]
I would like to get elastalert to create TheHive alert for particular elastalerts. Totally agree that not all NIDS and HIDS alerts to create TheHive alerts but some absolutely should. I think I probably have the port wrong or the url? I've tried http,https,localhost,IP address, dns name. I've created a user in cortex and plugged that api key and then a user in TheHive and plugged that api key. Is there a place I can find an example of the old nids2TheHive alert config?

My alert config is:
alert:
- hivealerter

hive_connection:
  hive_host: https://localhost
  hive_port: 9400
  hive_apikey: fHPtUXRjiUg87werWjdnJVbYqLAw/IDd

hive_alert_config:
  title: '{rule[name]} -- {match[alert]}'
  type: 'external'
  source: 'SecurityOnion'
  description: '{match[message]}'
  severity: 2
  tags: ['elastalert, SecurityOnion']
  tlp: 3
  status: 'New'
  follow: True

The error in my /opt/so/log/elastalert/elastalert.log is: 
2020-11-07 17:44:00,527    ERROR                 root Uncaught exception running rule my_random_rule rules being observed 1 - 9245fy3a0: 'alert'

**Comment by weslambert** [comment, marked as answer]
Here is an example of how we were doing it when we were sending alerts to TheHive:

https://github.com/Security-Onion-Solutions/securityonion/blob/5b16a6542231877906cb5f2ec0032a5ab50fd43f/salt/elastalert/files/rules/so/suricata_thehive.yaml

You might not meed the Salt/Jinja syntax when writing your rule.

**Reply by wdpless** [reply]
That did it. Thank you!
